features/polyPinion/README.md

```diff
@@ -1,11 +1,3 @@
-# questionnaire-feature
+# polyPinion
 
-qapp as a feature
-
-## Usage
-
-```
-$ npm install
-$ npm run build
-$ npm run serve
-```
+polyPinion is a feature for general questionnaires.
```

Comment:
Nit: Why remove the usage part? Not important though.

Reply:
Not really working now; it applied to the old `qapp`. For starters, you would need to build the core to be able to actually make it work.